addons/binding/org.openhab.binding.dwdunwetter/README.md

```diff
@@ -0,0 +1,114 @@
+# DwdUnwetter Binding
+
+Binding to retrieve the Weather Warnings of the "Deutscher Wetterdienstes" from the [DWD Geoserver](https://maps.dwd.de/geoserver/web/ "DWD Geoserver").
```

Comment:
The URL part behind the space seems unnecessary?

```suggestion
Binding to retrieve the Weather Warnings of the "Deutscher Wetterdienstes" from the [DWD Geoserver](https://maps.dwd.de/geoserver/web/).
```

Reply:
Done